packages/vite/src/node/plugins/css.ts

```diff
@@ -3095,6 +3095,8 @@ const esMap: Record<number, string[]> = {
   2021: ['chrome85', 'edge85', 'safari14.1', 'firefox80', 'opera71'],
   // https://caniuse.com/?search=es2022
   2022: ['chrome94', 'edge94', 'safari16.4', 'firefox93', 'opera80'],
+  // https://caniuse.com/?search=es2023
+  2023: ['chrome110', 'edge110', 'safari16.4', 'firefox115', 'opera96'],
```

Comment:
```suggestion
  2023: ['chrome110', 'edge110', 'safari16.4', 'opera96'],
```

Firefox doesn't have complete es2023 support yet, so I guess we need to remove it for now.

Reply:
OK, I have updated it. However, I have a question: How will removing Firefox affect the results?

Reply:
I think the main effect is that lightningcss won't take into account of firefox when transpiling the CSS. So if a CSS syntax is used that only firefox doesn't support, it won't try to transpile it down to something it supports.

Reply:
Thanks! ❤️